### ISSUE TITLE
APRS Object and item report support

### ISSUE THREAD

**Issue by kj4rdp** [ge0rg/aprsdroid#230]
#featurerequest To expand functionality of the amazing aprsdroid app will it be possible to allow aprs object and item reports using the ; or ) starting character along with the existing station position report?

**Comment by ge0rg** [comment]
There are no short- or mid-term plans to do that. It should be however possible to make a dedicated APRS object/item transmitter app based on the [APRSdroid API](https://aprsdroid.org/api/).

It might even be possible to have something like a simple(?) Tasker script to generate an object at your own location.

**Comment by omgitsgela** [comment]
That's kind of a shame. With such a popular application, you'd think more of the APRS specification would be covered. I guess event organizers will have to use APRSISCE to generate objects.

Will APRSDroid at least render objects on the map if someone else broadcasts them?

**Comment by ge0rg** [comment]
> That's kind of a shame.

I tend to agree, but sending objects is something only needed by a very small portion of users (and can lead to many issues on the air if used inappropriately), so this is a low priority feature for the app.

> Will APRSDroid at least render objects on the map if someone else broadcasts them?

Yes, that part should be working without problems.

**Comment by rubberduck203** [comment]
@ge0rg is that something you’d accept a PR for?

**Comment by na7q** [comment]
> [@ge0rg](https://github.com/ge0rg) is that something you’d accept a PR for?

I certainly would. Do you have something already done?

**Comment by rubberduck203** [comment]
No @na7q. I don’t like to spend effort building things that don’t have a chance of being merged. If an implementation would be considered, I can find time to give it a shot.

**Comment by na7q** [comment]
I think that's silly. Whether it gets merged or not doesn't really matter. What matters is that someone else can use it. I would love to see this implemented, it's just a difficult approach to doing it.

**Comment by rubberduck203** [comment]
@na7q I don’t have the time to maintain a fork. If the feature is DOA, then I can spend my effort elsewhere. You can think it’s silly to protect my precious off hours. I have no interest in debating that with you. 

What I’m interested in is knowing if this feature would be considered, if someone were to implement it.

**Comment by na7q** [comment]
You will spend your effort wherever you wish, nobody is demanding anything of you. You'll either do it or you won't. It's not up to us. We just throw out ideas, and someone does it or doesn't.

All features are considered in my fork if they don't cause issues. I would implement them into my fork if it works and doesn't break everything.

**Comment by git-pt-andy** [comment]
Our rural county is expanding our use of APRS in providing disaster situation assessment.  Being able to place objects would be very helpful. Thanks for considering. K7AHS
